### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kotenok2000**

### CODE REVIEW

**Comment by pfrazee** [comment]
NPM should be installed already when you installed node.

**Comment by millette** [comment]
On debian npm (1.4.21) must be installed separately: https://packages.debian.org/stretch/amd64/nodejs/filelist - but then again, that node version is very old (4.8.2).

Personally, I use n and [n-install](https://github.com/mklement0/n-install) but a lot of people are using [nvm](https://github.com/creationix/nvm).

**Comment by pfrazee** [comment]
Yeah I'll add some instructions about installing with nvm or n.

**Comment by paul90** [comment]
There are instructions for installing node using the package manager, on debian and ubuntu, available at https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions using the NodeSource distributions repository.

**Comment by Butterfly** [comment]
The timing of this thread has been WONDERFUL. A QUICK intro is I've apparently had my Github account since approximate 2009, but I am just now actually using it for almost the very first time. Am an absolute newbie learning on the fly here so my apologies in advance for beginner "oopsies" :smile:

A couple hopefully appropriate observations: #1 https://github.com/creationix/nvm#install-script worked perfect on Debian Buster. nvm then tried to (manually at my prompting) download node twice.

It failed both times most likely because dialup kept cutting off. nvm (understandably) started over at 0% download of the archive file. I was hoping to find flags documented that would circumvent that but so far am having hard time finding documentation that I can comprehend. Why I bother seeking a flag similar to wget's "-c" is that failed instances of that nature are about repeat wear and tear on the volunteer host servers and not just end user time spent.

#2 Nvm found more current than I did manually. Most current I could find was node-v6.11.2.tar.gz. Nvm chose the seemingly appropriate node-v8.3.0-linux-x64.tar.xz so I let it do what it thought should be done.

NOW after multiple nvm fails, I'm back at node-6.x k/t @paul90 pointing out node.js download page. I'd already been on that page, but this time the instructions made sense. I am working on the a-sumption that curl/apt-get's chosen version is correct for Beaker. PS Is my first successful curl. :smile:

Why I'm even writing is because you are considering potential readme notes. One "possible show stopper" glitch I encountered is that somewhere about 2 days ago I "apt-get install nodejs" directly. That was because "apt-get search node.js" was available via my one-liner Buster sources.list repository choice. That decision was not an informed one but was made because node.js popped up several times in online npm/nvm/node tips.

The notable "glitch" I encountered is that the curl/"apt-get install" route suggested via the @paul90 recommendation... *reinstalled* node.js after I had JUST installed it. ? :smile: Seems safe to think it was maybe about host repositories catching up to speed with the product's creators.

*EXCEPT THAT*: A further glitch (bug?) that has now been corrected is that npm did NOT exist when I installed node.js directly via Debian Buster (with no curl first). Some more research turned up "yum" with some possible potential, but that's as close as I could get.

The bottom line (for now): That "curl/apt-get install" reinstallation route JUST finished... and now the apparently important npm is available!! Thank you!! I have no idea what step comes next because it's been so long in between attempts, but I'm off to try anyway. :smile:

**Comment by flaketill** [comment]
Paul Frazee 

Great work and i'd like contributing with a package to Arch linux

I built a packages using the Arch Linux and yaourt, it's to don't have building from source if you aren't developer and install it on arch linux more easy:

You can view how to install on:
https://goo.gl/yaSXp7

If you want more information, please visit:

https://aur.archlinux.org/packages/beakerbrowser/

**Comment by pfrazee** [comment]
Cool, thanks @flaketill.
